README.Rmd

```diff
@@ -69,6 +69,14 @@ rmarkdown::draft(
 rmarkdown::render("my_preregistration.Rmd")
 ```
 
+## Uploading your preregistration
+After knitting your preregistration to a PDF file using this package, you need to upload this protocol to a trustworthy repository to complete your preregistration.
```

Comment:
Hmm, do you think we could say "you may upload the protocol"? In principle you could also do "in-house" preregistrations etc., right?